src/Controller/ApiClinicController.php

```diff
@@ -17,25 +17,17 @@
 
 class ApiClinicController extends AbstractController
 {
-    public function index()
-    {
-        return new JsonResponse(['status' => 'ok']);
-    }
-
     public function list()
     {
-        $clinics = $this->getDoctrine()
-            ->getRepository('App\Entity\Clinic')
+        $clinics = $this->getDoctrine()->getRepository(Clinic::class)
```

Comment:
In the near future, I'll teach you how you do to inject this repository on this controller instead of trying to get it by this way. When you get a change to read [this article](https://www.tomasvotruba.cz/blog/2017/10/16/how-to-use-repository-with-doctrine-as-service-in-symfony/), please.